File: apps/auth-server/src/main/java/com/example/chat/auth/server/api/dto/request/CompleteMfaRequest.java
package com.example.chat.auth.server.api.dto.request;

import jakarta.validation.constraints.NotBlank;

/**
 * MFA 완료 요청 DTO
 */
public record CompleteMfaRequest(
        @NotBlank(message = "mfaToken cannot be blank") String mfaToken,
        @NotBlank(message = "mfaSessionId cannot be blank") String mfaSessionId,
        @NotBlank(message = "mfaMethod cannot be blank") String mfaMethod,
        @NotBlank(message = "otpCode cannot be blank") String otpCode) {
}

File: apps/auth-server/src/main/kotlin/com/example/chat/auth/server/api/dto/request/CompleteMfaRequest.kt
package com.example.chat.auth.server.api.dto.request

import jakarta.validation.constraints.NotBlank

/** MFA 완료 요청 DTO */
data class CompleteMfaRequest(
	@field:NotBlank(message = "mfaToken cannot be blank")
	val mfaToken: String? = null,

	@field:NotBlank(message = "mfaSessionId cannot be blank")
	val mfaSessionId: String? = null,

	@field:NotBlank(message = "mfaMethod cannot be blank")
	val mfaMethod: String? = null,

	@field:NotBlank(message = "otpCode cannot be blank")
	val otpCode: String? = null

)
